docs: mismatch between explanation and code in triggers (#33783)
github/docs: content/actions/publishing-packages/publishing-docker-images.md

## Publishing images to Docker Hub

> Each time you create a new release on {% data variables.product.product_name %}, you can trigger a workflow to publish your image. The workflow in the example below runs when the `release` event triggers with the `published` activity type. For more information on the `published` event, see "[AUTOTITLE](/actions/using-workflows/events-that-trigger-workflows#release)."

Reviewer: We could omit the link here:
```suggestion
Each time you create a new release on {% data variables.product.product_name %}, you can trigger a workflow to publish your image. The workflow in the example below runs when the `release` event triggers with the `published` activity type.
```

{% data reusables.package_registry.container-registry-ghes-beta %}
{% endif %}

> Each time you create a new release on {% data variables.product.product_name %}, you can trigger a workflow to publish your image. The workflow in the example below runs when the `release` event triggers with a push trigger. For more information on the `release` event, see "[AUTOTITLE](/actions/using-workflows/events-that-trigger-workflows#release)."

Reviewer: Slight correction here, and again I think we can omit the link.

```suggestion
Each time you create a new release on {% data variables.product.product_name %}, you can trigger a workflow to publish your image. The workflow in the example below runs when a change is pushed to the `release` branch.
```
